clients/src/main/resources/common/message/BrokerRegistrationRequest.json

```diff
@@ -51,7 +51,7 @@
     },
     { "name": "Rack", "type": "string", "versions": "0+", "nullableVersions": "0+",
       "about": "The rack which this broker is in." },
-    { "name": "IsMigratingZkBroker", "type": "int8", "versions": "0+", "taggedVersions": "0+", "tag": 0, "ignorable": true,
+    { "name": "IsMigratingZkBroker", "type": "bool", "versions": "0+", "taggedVersions": "0+", "tag": 0, "ignorable": true,
```

Comment:
see my comment in the main PR conversation thread: this should not be tagged and we need a new version of the rpc ...